### PR TITLE
#92 timeout info volume - fargo & unstable

### PR DESCRIPTION
**Pull request by kinvaris**

### CODE REVIEW

**Comment by kinvaris** [comment]
When a `info_volume` call times out it will be added to the `halted` volumes: `[FAILED] Detected volumes that are HALTED in vPool 'myvpool10': 3d4f2b00-9a11-44a7-ace5-dbd88269acef, 4d7bb9ae-914f-4ad3-b8bb-476759ac0943, 6769467b-8e45-43c1-8642-197d6da4a340, 9ad8a87d-e6aa-4efc-b672-7d0f83354bd2, af9fef3c-5e73-4d9d-8dfc-60f8b5fd07bb`

**Comment by kinvaris** [comment]
#92

File: ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py [hunk 1]
```diff
@@ -15,11 +15,11 @@
 #
 # Open vStorage is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY of any kind.
-
+import re
 import os
 import grp
 import glob
-import re
+import time
```
Comment (JeffreyDevloo) [inline ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py]: Unused import

Comment (kinvaris) [inline ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py]: This was to simulate the timeout, forgot to remove

**Pull request merged**